lsp/biome.lua

```diff
@@ -46,11 +46,9 @@ return {
     -- manager lock file.
     local root_markers = { 'package-lock.json', 'yarn.lock', 'pnpm-lock.yaml', 'bun.lockb', 'bun.lock', 'deno.lock' }
     -- Give the root markers equal priority by wrapping them in a table
-    root_markers = vim.fn.has('nvim-0.11.3') == 1 and { root_markers } or root_markers
-    local project_root = vim.fs.root(bufnr, root_markers)
-    if not project_root then
-      return
-    end
+    root_markers = vim.fn.has('nvim-0.11.3') == 1 and { root_markers, { '.git' } } or root_markers
```

Comment:
Nice use of the priority grouping. To avoid confusion for users on 0.11.2 or older, should we do this:
```suggestion
    root_markers = vim.fn.has('nvim-0.11.3') == 1 and { root_markers, { '.git' } } or vim.list_extend(root_markers, {'.git'})
```